### PR TITLE
Blog post about new LSP4IJ

### PR DESCRIPTION
**Pull request by fbricon**
Signed-off-by: Fred Bricon <fbricon@gmail.com>

### CODE REVIEW

**Comment by fbricon** [comment]
Do not merge until Jetbrains approved the LSP4IJ 0.0.2 release

**Review by rgrunber — APPROVED** [review]

**Pull request merged**
